src/v/kafka/protocol/schemata/delete_topics_request.json

```diff
@@ -20,10 +20,20 @@
   // Versions 0, 1, 2, and 3 are the same.
   //
   // Version 4 is the first flexible version.
-  "validVersions": "0-4",
+  //
+  // Version 5 adds ErrorMessage in the response and may return a THROTTLING_QUOTA_EXCEEDED error
+  // in the response if the topics deletion is throttled (KIP-599).
+  //
+  // Version 6 reorganizes topics, adds topic IDs and allows topic names to be null.
+  "validVersions": "0-6",
   "flexibleVersions": "4+",
   "fields": [
-    { "name": "TopicNames", "type": "[]string", "versions": "0+", "entityType": "topicName",
+    { "name": "Topics", "type": "[]DeleteTopicState", "versions": "6+", "about": "The name or topic ID of the topic",
+      "fields": [
+        {"name": "Name", "type": "string", "versions": "6+", "nullableVersions": "6+", "default": "null", "entityType": "topicName", "about": "The topic name"},
+        {"name": "TopicId", "type": "uuid", "versions": "6+", "about": "The unique topic ID"}
+    ]},
+    { "name": "TopicNames", "type": "[]string", "versions": "0-5", "entityType": "topicName", "ignorable": true,
```

Comment:
why are we adding version 6 if we are only bumping to version 5? i guess it doesn't affect anything, but is there something i'm missing?

Reply:
This is because I am copying the schema from upstream kafka, character for character. I don't want to make manual changes to this file. It won't matter anyway because the request version upper bound is set to 5 in `delete_topics.h`